pkg/misc/uuid/snowflake.go

```diff
@@ -49,14 +51,14 @@ func NewWorker() (*SnowflakeWorker, error) {
 }
 
 func (w *SnowflakeWorker) NextID() (int64, error) {
-	ts := time.Now().UnixMilli()
+	ts := misc.CurrentTimeMillis()
```

Comment:
```suggestion
	ts := misc.CurrentTimeMilliSec()
```

Reply:
there is no misc.CurrentTimeMilliSec()，suggest update at next pr